drivers/misc/optee.c

```diff
@@ -0,0 +1,648 @@
+/****************************************************************************
+ * drivers/misc/optee.c
+ *
```

Comment:
Recommend to add a 1-line description of what is OP-TEE, since it is not obvious.